Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/view/MultiImageView.java

```diff
@@ -88,7 +88,7 @@
 
     private Context context;
     private ImageView playView;
-    private GestureDetector doubleTapDetector = new GestureDetector(context, new GestureDetector.SimpleOnGestureListener() {
```

Comment:
Why?

Reply:
context is always null, so I figured I'd just replace it. Will revert.

Reply:
That's weird, context should be the context of the Activity, namely StartActivity.